frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Web Terminal for Developer user
+    As a user with admin rights, I should be able to use web terminal
+
+
+Background:
+    Given user has logged in as admin user
+    And user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-admin"
+    And user is at the Topolgy page
```

Comment:
Why does the page matter?

Reply:
In the background, it doesn't matter. I will remove it.